src/utils/env.rs

```diff
@@ -9,6 +9,7 @@ pub struct Environment {
     #[serde(default = "default_beacon_node_rpc")]
     pub beacon_node_rpc: String,
     #[serde(default = "default_logger")]
+    #[allow(dead_code)] // Temporal until we move to tracing
```

Comment:
this one's just a tad ugly but as i'll be replacing the logger for tracing soon it's ok—just want CI to pass